common.css

```diff
@@ -0,0 +1,5 @@
+body{
```

Comment:
also use * selector to reset margin and padding, you can create reset.css where you can have it